dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -55,12 +55,12 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	cp -R /sources-3/grit-b033674/* externals/grit
+	cp -R /$sourceDir3/grit-b033674/* externals/grit
 
 	export GYP_DEFINES="
-		gtest_dir='/sources-2/googletest-release-1.8.0/googletest/include'
-		gtest_src_dir='/sources-2/googletest-release-1.8.0/googletest'
-		rapidjson_dir=/system/develop/headers
+		gtest_dir='/$sourceDir2/googletest-release-1.8.0/googletest/include'
+		gtest_src_dir='/$sourceDir2/googletest-release-1.8.0/googletest'
```

Comment:
Oops, you forgot to drop the / in front of `$sourceDir2` and `$sourceDir3`. These variables are already absolute paths.